### DISCUSSION TITLE
Vue: No overload matches this call. The last overload gave the following error.

### DISCUSSION THREAD

**Discussion by asasas234** [wxt-dev/wxt#1259] [Q&A] [answered]
<img width="826" alt="image" src="https://github.com/user-attachments/assets/712380da-3667-4480-9feb-d5268e459491">

I keep encountering this error despite following the official documentation and using the latest version.

**Comment by aklinker1** [comment]
Vitest hasn't added vite 6 support yet, at least as of a few days ago. How many different versions of vite are installed?

The quick and dirty solution is to delete your lock file and recreate it. Other solution is to manually ensure only one version is installed.

**Reply by asasas234** [reply]
@aklinker1 I confirmed that only one version of vite is installed, and the problem still persists.

**Reply by aklinker1** [reply]
What version?

**Reply by asasas234** [reply]
@aklinker1 I found that I have version 6.0.0 installed. Which version do you recommend?

**Reply by aklinker1** [reply, marked as answer]
Does `^5.0.0` work?

**Reply by asasas234** [reply]
@aklinker1 I understand now, it turns out that I rely on wxt version 6.0, but vitest depends on version 5.4.11. I didn't explicitly declare the dependency on vite before, but now that I explicitly declare the dependency on vite, it works fine.
